### ISSUE TITLE
Uninstall app doesn't clean files

### ISSUE THREAD

**Issue by lorenzoPrimi** [ooni/probe#1257]
Uninstalling the desktop app and reinstalling it led to keep the past test results.
It is surprising how osx doesn’t clean app files, usually I use 3rd party apps like Clean my mac to clean general garbage of uninstalled apps.
We need to investigate if we can do some sort of cleanup or at least tell the user.

**Comment by bassosimone** [comment]
Discussed in today's fronted meeting. We reviewed existing app patterns for macOS and Windows. On macOS it's pretty common for apps, e.g. VSCode, to not cleanup the disk space they are using inside the `~/Library` directory. We are using the pattern of writing inside `~/Library` though, which is compatible with the established pattern of zapping apps using https://www.appzapper.com/. Something like this also happens on Windows, and I have just checked we're (correctly) using `AppData\\Local` and `AppData\\Roaming`.

The underlying problems seems how to ensure that we're not consuming too much disk space. We will prioritise work to check whether a measurement was correctly submitted so we don't need to keep it on disk https://github.com/ooni/probe-engine/issues/948. Also, @hellais is going to think about improving the UX around cancelling measurements.
